tests/utils_test.py

```diff
@@ -2,11 +2,7 @@
 
 from __future__ import annotations
 
-import os
-from collections.abc import Iterator
-from unittest.mock import patch
-
-import pytest
+from _pytest.monkeypatch import MonkeyPatch
```

Comment:
Underbar pytest?  What's that about?

Reply:
This is just for typing. pytest provides the monkeypatch decorator, which declares a MonkeyPath object, but they don't consider the object to be part of the public API (even though it is, indirectly, via the fixture) and only expose a type for it in their underbar module. It's just a mypy thing so that it won't complain; you have to do the same thing for the log capture fixture when testing logging.

However, thank you for asking, because apparently both of those have now been fixed! Somehow I missed that in 6.2.0 a bunch of the relevant types were properly exported, and we can stop using `_pytest` for typing imports now. That's now fixed here as well.